.github/workflows/docker_build_push.yml

```
@@ -222,3 +235,15 @@ jobs:
         name: Add PR-specific tags
         run: |
           gcloud artifacts docker tags add '${{ env.IMAGE_NAME }}:${{ github.sha }}' '${{ format('{0}:pr-{1}', env.IMAGE_NAME, github.event.number) }}'
+
+      - name: Create Sentry release
+        if: ${{ inputs.create_sentry_release }}
+        uses: getsentry/action-release@v1.4.1
```

Bump to [latest version 1.7.0](https://github.com/getsentry/action-release/releases/tag/v1.7.0) which uses sentry cli 2.24.1 - [2.0 of the cli](https://github.com/getsentry/sentry-cli/releases/tag/2.0.0) has been out for some time and looks like there's no issues with adopting it
```suggestion
        uses: getsentry/action-release@v1.7.0
```